src/api/models/queryBody.ts

```diff
@@ -10,7 +10,7 @@ export interface QueryBody<
   readonly count: boolean;
```

Comment:
Typo in doc `/** Whether to count **or** items that satisfy given query or not. */`.
Also property name is unclear, I suggest naming it 'isCountItems' or maybe `doCountItems`.